lib/pf/authentication.pm

```diff
@@ -245,15 +246,33 @@ If source_id_ref is defined then it will be set to the matching source_id
 
 =cut
 
+our %ALLOW_ACTIONS_TYPES = (
+    $Actions::MARK_AS_SPONSOR  => {$Actions::MARK_AS_SPONSOR  => 1},
+    $Actions::SET_ACCESS_LEVEL => {$Actions::SET_ACCESS_LEVEL => 1},
+    $Actions::SET_ROLE         => {$Actions::SET_ROLE         => 1},
+    $Actions::SET_UNREG_DATE   => {
+        $Actions::SET_UNREG_DATE      => 1,
+        $Actions::SET_ACCESS_DURATION => 1,
+    }
+);
+
+our %ACTION_VALUE_FILTERS = (
```

Comment:
No because it would mean that pf::Authentication::constants would have to import pf::config